.changeset/silent-sloths-flow.md

```diff
@@ -1,6 +1,6 @@
 ---
-"@fuel-wallet/types": minor
-"@fuel-wallet/sdk": minor
+'@fuel-wallet/types': minor
+'@fuel-wallet/sdk': minor
```

Comment:
This will not be an issue for the changesets?

Reply:
why?

Reply:
Is more a question as I don't know how changeset cli parsers